src/test/TxTests.cpp

```diff
@@ -442,10 +449,31 @@ class TxSetFrameStrictOrderForTesting : public TxSetFrame
     void sortForHash() override{};
 };
 
+TxSetResultMeta
+closeLedger(Application& app, std::vector<TransactionFrameBasePtr> const& txs,
+            bool strictOrder)
+{
+    auto lastCloseTime = app.getLedgerManager()
```

Comment:
I tried this, but there were a bunch of test failures. I thought it would be better to leave as is instead of spending the time it would take to fix all of them.